src/backend/dx12/src/device.rs

```diff
@@ -1528,8 +1530,9 @@ impl d::Device<B> for Device {
         //     ...
         //     DescriptorTable0: Space: 1 (+1) (SrvCbvUav)
         //     Root Descriptors
-        //     DescriptorTable0: Space: 2 (+1) (Sampler)
-        //     DescriptorTable1: Space: 3 (+1) (SrvCbvUav)
+        //     DescriptorTable1: Space: 2 (+1) (SrvCbvUav)
+        //     DescriptorTable0: Space: 3 (+1) (Sampler)
+        //     Root Descriptors
```

Comment:
should be slightly different given the current layout (incl. changes to sampler space assignment which hasn't be reflected so far)
```
DescriptorTable0: Space: 1 (SrvCbvUav)
DescriptorTable0: Space: 1 (Sampler)
Root Descriptors 0
DescriptorTable1: Space: 2 (SrvCbvUav)
RootDescriptors 1
```

Reply:
Thanks! I'll update that comment.